pom.xml

```diff
@@ -357,6 +357,11 @@
 			<artifactId>hibernate-jpa-2.0-api</artifactId>
 			<version>1.0.1.Final</version>
 		</dependency>
+		<dependency>
```

Comment:
shouldn't this dependency be placed in openhospital-api? I assume that it's used to validate  endpoint input parameters. And if we need javax.validation annotations somewhere in code using validation-api was better option than concrete implementation

Reply:
I've tried to comment it in `core` and annotations like `@NotNull` into models are no longer resolved